packages/backend/src/open_payments/payment/incoming/errors.ts

```diff
@@ -24,6 +26,18 @@ export const errorToCode: {
   [IncomingPaymentError.InactiveWalletAddress]: 400
 }
 
+export const errorToCode: {
+  [key in IncomingPaymentError]: string
+} = {
+  [IncomingPaymentError.UnknownWalletAddress]: GraphQLErrorCode.NotFound,
+  [IncomingPaymentError.InvalidAmount]: GraphQLErrorCode.BadUserInput,
+  [IncomingPaymentError.UnknownPayment]: GraphQLErrorCode.NotFound,
+  [IncomingPaymentError.InvalidState]: GraphQLErrorCode.BadUserInput,
+  [IncomingPaymentError.InvalidExpiry]: GraphQLErrorCode.BadUserInput,
+  [IncomingPaymentError.WrongState]: GraphQLErrorCode.BadUserInput,
```

Comment:
Perhaps Conflict/409 would be a better match? The docs mention using it when the resource is in the wrongs state https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

```suggestion
  [IncomingPaymentError.WrongState]: GraphQLErrorCode.Conflict,
```

also matches how we map it here: https://github.com/interledger/rafiki/blob/main/packages/backend/src/open_payments/payment/incoming/errors.ts#L23